tests/test_storage.py

```diff
@@ -1092,6 +1099,53 @@ def test_delete_single_submission_or_reply_race_guard(homedir, mocker):
     mock_remove.call_count == 1
 
 
+def test_delete_single_submission_or_reply_single_file(homedir, mocker):
+    """
+    This test checks that calling the delete_single_submission_or_reply
+    method deletes the file as well as the folder it's inside.
+    """
+
+    source = factory.Source(journalist_designation="dissolved-steak")
+    file_server_filename = '1-dissolved-steak-msg.gpg'
+    test_obj = db.File(
+        source=source, uuid="test", size=123, filename=file_server_filename,
+        download_url='http://test/test')
+    source_directory = os.path.join(
```

Comment:
Here again you could use `File.location`, unless you want this to be a canary for changes to that function. :slightly_smiling_face: